### PR TITLE
feat: add `workspace-path` parameter

### PR DESCRIPTION
**Pull request by Hemant11111**
Fix issue in generating coverage report for monorepo. https://github.com/barecheck/code-coverage-action/issues/235

### CODE REVIEW

File: src/input.js [hunk 1]
```diff
@@ -38,6 +38,8 @@ const getBaseLcovFile = () => valueOrFalse(core.getInput("base-lcov-file"));
 const getSendSummaryComment = () =>
   valueOrFalse(core.getInput("send-summary-comment"));
 
+const getWorkspacePath = () => core.getInput("workspace-path");
```
Comment (vitaliimelnychuk) [inline src/input.js:41]: Can you add please this parameter to Readme file?

File: src/services/changedFilesCoverage.js [hunk 2]
```diff
@@ -16,17 +17,19 @@ const getChangedFilesCoverage = async (coverage) => {
     pullNumber
   });
 
+  const workspacePath = getWorkspacePath();
   const changedFilesCoverage = coverage.data.reduce(
     (allFiles, { file, lines }) => {
+      const filePath = workspacePath ? `${workspacePath}/${file}` : file;
```
Comment (vitaliimelnychuk) [inline src/services/changedFilesCoverage.js:23]: Ideally, would be good to use https://www.w3schools.com/nodejs/met_path_join.asp

As users can enter the workspace with and without `/`

Comment (vitaliimelnychuk) [reply src/services/changedFilesCoverage.js:23]: Also the action can be run in Linux/Windows machines theoretically

File: src/input.js (hunk 1 above)
Comment (Hemant11111) [reply src/input.js:41]: @vitaliimelnychuk updated the readme

File: src/services/changedFilesCoverage.js (hunk 2 above)
Comment (Hemant11111) [reply src/services/changedFilesCoverage.js]: @vitaliimelnychuk That Makes sense. using path.join() to merge the paths now.

**Review by vitaliimelnychuk — APPROVED** [review]
Looks good, I'll generate new release today and I let you know

**Pull request merged**